recipes-core/disk-encryption/init

```diff
@@ -25,6 +30,34 @@ log() {
     echo "$1" > /var/volatile/system-api.fifo
 }
 
+find_target_disk() {
+    local target_path
+    
+    # First try to find disk by LUN
+    target_path=$(readlink -f "/dev/disk/by-path/"*"scsi-0:0:0:${TARGET_LUN}" 2>/dev/null || true)
```

Comment:
```suggestion
    target_path=$(readlink -f /dev/disk/by-path/*scsi-0:0:0:${TARGET_LUN} 2>/dev/null || true)
```

Reply:
Is there a reason to remove the quotes?
I've tested the command manually with quotes and it worked with them too.